[2024-08-04T06:59:04.123+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-04T06:59:04.155+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_process.etl_process.load_processed_data scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T06:59:04.164+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_process.etl_process.load_processed_data scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T06:59:04.165+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-04T06:59:04.179+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): etl_process.load_processed_data> on 2024-08-03 00:00:00+00:00
[2024-08-04T06:59:04.187+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_process', 'etl_process.load_processed_data', 'scheduled__2024-08-03T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpgl_tht4e']
[2024-08-04T06:59:04.190+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask etl_process.load_processed_data
[2024-08-04T06:59:04.191+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=120) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-04T06:59:04.192+0000] {standard_task_runner.py:63} INFO - Started process 137 to run task
[2024-08-04T06:59:04.246+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_process.etl_process.load_processed_data scheduled__2024-08-03T00:00:00+00:00 [running]> on host 28819dc9564a
[2024-08-04T06:59:04.333+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ztamnaja' AIRFLOW_CTX_DAG_ID='etl_process' AIRFLOW_CTX_TASK_ID='etl_process.load_processed_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-03T00:00:00+00:00'
[2024-08-04T06:59:04.335+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-04T06:59:04.442+0000] {base.py:84} INFO - Using connection ID 'destination_db_conn' for task execution.
[2024-08-04T06:59:04.454+0000] {logging_mixin.py:188} INFO - Successfully inserted/updated data in dim_products
[2024-08-04T06:59:04.458+0000] {logging_mixin.py:188} INFO - Successfully inserted/updated data in dim_customers
[2024-08-04T06:59:04.464+0000] {logging_mixin.py:188} INFO - Successfully inserted/updated data in fact_sales
[2024-08-04T06:59:04.465+0000] {logging_mixin.py:188} INFO - All data inserted/updated successfully
[2024-08-04T06:59:04.467+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-04T06:59:04.468+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-04T06:59:04.478+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_process, task_id=etl_process.load_processed_data, run_id=scheduled__2024-08-03T00:00:00+00:00, execution_date=20240803T000000, start_date=20240804T065904, end_date=20240804T065904
[2024-08-04T06:59:04.528+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-04T06:59:04.553+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-04T06:59:04.556+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
